### DISCUSSION TITLE
🛫 Version: 1.7.2

### DISCUSSION THREAD

**Discussion by valnoxy** [valnoxy/t440p-oc#51] [Announcements]
Hey 👋,
I hope you all have a great day!

# Changes
OpenCore was updated to Version ```0.7.8```.

The following kexts was added:
   - VoodooRMI (issue #27)

The following kexts was updated:
  - AppleALC to ```1.6.9```
  - FeatureUnlock to ```1.0.6```
  - Lilu to ```1.6.0```
  - VoodooPS2Controller to ```2.2.7```
  - WhateverGreen to ```1.5.7```

The following changes was made to the config:
- Disabled Kext ```VoodooInput.kext```

<hr /><em>This discussion was created from the release <a href='https://github.com/valnoxy/t440p-oc/releases/tag/v1.7.2'>🛫 Version: 1.7.2</a>.</em>

**Comment by faizanafz** [comment]
Hey valnoxy, i have installed macOS Monetery on my Lenovo Thinkpad T440p it's working perfectly, so thanks a lot for sharing this guide.

But I have received this error:

Volume Hash Mismatch
Hash mismatch detected on volume disk1s5.
macOS should be reinstalled on this volume.

Here is a screenshot of the error: https://user-images.githubusercontent.com/55034706/158290686-e42ae21f-e34f-46bf-885a-1fbf64c6c027.jpeg

Even after getting this error, everything on my Laptop works just fine. But a person on the web told me that "I should resolve this issue" & gave me this troubleshooting option.

- Turn on Laptop or restart
- Enter macOS Recovery by holding Windows key & R at the same time till the Apple Logo shows up
- Go to Disk Utility in macOS Recovery
- Unmount everything below "Container disk 1" & run First Aid
- Restart/Reboot the Laptop

I have tried this troubleshooting option but I'm not sure / can't confirm wether the problem has been resolved or not.
I have also not received any notification regarding that error.

On my Disk Utility nothing has changed either. https://user-images.githubusercontent.com/55034706/158293012-fcc39dc4-5c3a-49ee-8d29-d90384d4ecd4.png

It says "Mount' but when I click on it nothing happens. But I can mount it through a Software called "ESP Mounter Pro"

Download Link of ESP Mounter Pro: https://olarila.com/files/Utils/ESP%20Mounter%20Pro.app_v1.9.1.zip

Here's a screenshot after I mount the partition through ESP Mounter Pro: https://user-images.githubusercontent.com/55034706/158294068-d06972ab-d4e0-41ee-9c2d-fdb24f2c06f7.png


Please help me resolve this issue, i would really appreciate it.

Best Regards,
Faizan Afzal

**Comment by centraljava** [comment]
hello, wonderful work!

is this system able to support Ventura?
tq

**Comment by PanyankazKyeva** [comment]
good afternoon. I wanted to try installing MacOs, but when I enter a command in the terminal to download a file, it says 'python' and nothing else. I don't know much about this stuff, so I thought I'd ask for help. I also tried ubuntu, but the result was about the same, could someone please tell me what to do to make it download and install properly. There is a suggestion that maybe need to update python.
![image](https://github.com/valnoxy/t440p-oc/assets/109184146/549dafd9-85a7-442b-9aa2-f985f8b623c0)
